ci-operator/jobs/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master-presubmits.yaml

```diff
@@ -64,6 +64,72 @@ presubmits:
         secret:
           secretName: sentry-dsn
     trigger: (?m)^/test( | .* )e2e-aws,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    context: ci/prow/e2e-aws
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+    name: pull-ci-openshift-ovn-kubernetes-master-e2e-aws-ovn-kubernetes
+    rerun_command: /test e2e-aws
```

Comment:
`/test e2e-aws-ovn-kubernetes`
(We're going to have to call it `e2e-aws-ovn-kubernetes` rather than `e2e-aws` in cluster-network-operator, and I think it's best to keep the same name everywhere, and just not have any `e2e-aws` in openshift/ovn-kubernetes.)
(There are a bunch of other places here where you have to change `e2e-aws` to `e2e-aws-ovn-kubernetes` too)
(Applies to the CNO config change too.)